proto/src/constraints.proto

```diff
@@ -32,10 +34,10 @@ message SummaryConstraintMsg {
   oneof second {
      string second_field = 3;
      double value = 4;
-     SummaryBetweenConstraintMsg between = 5;
+     SummaryBetweenConstraintMsg between = 7;
   }
-  Op op = 6;
-  bool verbose = 7;
+  Op op = 5;
+  bool verbose = 6;
```

Comment:
We want to make sure these are backwards compatible.

Reply:
This in fact is a fix for: https://github.com/whylabs/whylogs/commit/f90b45ae53d50ffde66beec8f4e6184061beef32#diff-3ead279fdd02046f469418a748586aa737b3a3cf99be981b99db1bee165a2a22R38
So now editing them back like they previously were.

Reply:
ahh makes sense. While constraints are a new feature, we want to make sure we don't break deserializing an older protobuf serialized filed, thanks for fixing it back to how they previously were.